docs/references/backend/overview.mdx

```diff
@@ -4,97 +4,196 @@ description: The JavaScript Backend SDK exposes Clerk's backend API resources an
 
 # JavaScript Backend SDK
 
-The [JavaScript Backend SDK](https://github.com/clerk/javascript/tree/main/packages/backend) exposes Clerk's backend API resources and low-level authentication utilities **for JavaScript environments**. While the backend SDK is mainly used as a building block for Clerk's higher-level SDKs, it can also be used on its own for advanced flows and custom integrations.
+Clerk's JavaScript Backend SDK exposes [Clerk's Backend API](https://clerk.com/docs/reference/backend-api) resources and low-level authentication utilities **for JavaScript environments**.
 
-Clerk SDKs expose an instance of the Backend SDK for use in server environments.
+For example, if you wanted to get a list of all users in your application, instead of creating a fetch to Clerk's [`https://api.clerk.com/v1/users`](https://clerk.com/docs/reference/backend-api/tag/Users#operation/GetUserList) endpoint, you can use the [`users.getUserList()`](/docs/references/backend/user/get-user-list) method provided by the JavaScript Backend SDK.
```

Comment:
@alexisintech just to be clear the `users.getUserList()` is not exposed directly from the `@clerk/backend` just like you mention in the docs below.

Reply:
it reads "All resource operations are mounted as sub-APIs on the `clerkClient` object. For example, if you would like to get a list of all of your application's users, you can use the `getUserList()` method on the `users` sub-API. You can find the full list of available sub-APIs and their methods in the sidebar."

the paragraph begins with stating resources are mounted as sub-APIs on `clerkClient` so users.getUserList() is mounted on `clerkClient`

is this the section you meant?